src/slack/slack.js

```diff
@@ -607,11 +607,11 @@ export class Slack {
     const userMessage = order.messages[0]
 
     if (userMessage) {
-      const {channel, ts} = order.orderConfirmation
+      const {originalMessageInfo} = order
       try {
-        await this.boltApp.client.chat.delete({channel, ts})
+        await this.boltApp.client.chat.delete(originalMessageInfo)
```

Comment:
actually, even though this looks clean, it hides what's inside `originalMessageInfo`. so please make it more transparent by e.g.:
```js
const {originalMessageInfo} = order
const {channel, ts} = originalMessageInfo
try {
  await this.boltApp.client.chat.delete({channel, ts})
```
or this should work too:
```js
const {originalMessageInfo: {channel, ts}} = order
try {
  await this.boltApp.client.chat.delete({channel, ts})
```